File: 3403_Find_the_Lexicographically_Largest_String_From_the_Box_I.py
class Solution(object):
    def answerString(self, word, numFriends):
        if numFriends == 1: return word
        res = ""
        k = len(word) - numFriends + 1
        for i in range(len(word)):
            res = max(res, word[i:i+k])
        return res

File: 3403_Find_the_Lexicographically_Largest_String_From_the_Box_I.cpp
class Solution {
public:
    string answerString(string x, int y) {
        if (y == 1) return x;
        string a;
        for (int j = 0; j < x.size(); ++j)
            a = max(a, x.substr(j, x.size() - y + 1));
        return a;
    }
};
